src/Microsoft.AspNet.Authentication.OAuth/OAuthHandler.cs

```diff
@@ -233,7 +173,7 @@ protected override Task HandleSignInAsync(SignInContext context)
             throw new NotSupportedException();
         }
 
-        protected override Task<bool> HandleForbiddenAsync(ChallengeContext context)
+        protected override Task HandleForbiddenAsync(ChallengeContext context)
         {
             throw new NotSupportedException();
```

Comment:
You don't need these overrides anymore, correct? They're handled in the new base class.